runtime/common/src/registrar.rs

```diff
@@ -563,6 +563,15 @@ impl<T: Trait> ActiveParas for Module<T> {
 pub struct LimitParathreadCommits<T: Trait + Send + Sync>(sp_std::marker::PhantomData<T>) where
 	<T as system::Trait>::Call: IsSubType<Module<T>, T>;
 
+impl<T: Trait + Send + Sync> LimitParathreadCommits<T> where
+	<T as system::Trait>::Call: IsSubType<Module<T>, T>
+{
+	/// Create a new `LimitParathreadCommits` struct.
+	pub fn new() -> Self {
+		LimitParathreadCommits(sp_std::marker::PhantomData)
```

Comment:
Same as above.

Reply:
Other things like `system::CheckWeight` are [created](https://github.com/paritytech/polkadot/pull/966/files#diff-42f5735696ee7d721d953f88bb1c5444R553) with `new()` so I decided to take the same approach to make the code look more uniform.

Reply:
Yeah in substrate we currently have some that implement `Default` but most just provide a `new` method.